FFXIVClientStructs/FFXIV/Client/UI/Misc/RaptureTextModule.cs

```diff
@@ -54,17 +54,106 @@ public unsafe partial struct RaptureTextModule {
+    public partial CStringPointer FormatAddonText2Int(uint addonId, int intParam);
+
+    [MemberFunction("E8 ?? ?? ?? ?? EB 51 0F B6 DB")]
     public partial CStringPointer FormatAddonText2IntInt(uint addonId, int intParam1, int intParam2);
 
-    [MemberFunction("E8 ?? ?? ?? ?? 48 8B D8 EB 38")] // FormatAddonText2<int,int,uint>
-    public partial CStringPointer FormatAddonText2IntIntUInt(uint addonId, int value1, int value2, uint value3);
+    [MemberFunction("E8 ?? ?? ?? ?? 48 8B D8 EB 38")]
+    public partial CStringPointer FormatAddonText2IntIntUInt(uint addonId, int intParam1, int intParam2, uint uintParam);
+
+    [MemberFunction("E8 ?? ?? ?? ?? EB 72 4C 8B 42 30")]
+    public partial CStringPointer FormatAddonText2IntIntUIntUInt(uint addonId, int intParam1, int intParam2, uint uintParam1, uint uintParam2);
+
+    [MemberFunction("E8 ?? ?? ?? ?? 8D 4D 64")]
+    public partial CStringPointer FormatAddonText2IntIntUIntUIntUInt(uint addonId, int intParam1, int intParam2, uint uintParam1, uint uintParam2, uint uintParam3);
+
+    [MemberFunction("48 89 5C 24 ?? 55 56 57 41 56 41 57 48 8B EC 48 83 EC 50 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 45 F8 48 8D B9 ?? ?? ?? ?? 48 8B F1 48 8B CF 45 8B F1 41 8B D8 44 8B FA E8 ?? ?? ?? ?? 33 C0 C6 45 F0 00 48 89 45 E0 48 8D 55 E0 48 8D 45 E0 89 5D E0 48 8B CF 48 89 45 E8 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 33 C0 C6 45 F0 00 48 89 45 E0 48 8D 55 E0 48 8D 45 E0 44 89 75 E0 48 8B CF 48 89 45 E8 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 33 C0 C6 45 F0 00 48 89 45 E0 48 8D 55 E0 8B 45 50 48 8B CF 89 45 E0 48 8D 45 E0 48 89 45 E8 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 33 C0 C6 45 F0 00 48 89 45 E0 48 8D 55 E0 8B 45 58 48 8B CF 89 45 E0 48 8D 45 E0 48 89 45 E8 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 33 C0 C6 45 F0 00 48 89 45 E0 48 8D 55 E0 8B 45 60 48 8B CF 89 45 E0 48 8D 45 E0 48 89 45 E8 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 33 C0 C6 45 F0 00 48 89 45 E0 48 8D 55 E0 8B 45 68 48 8B CF 89 45 E0 48 8D 45 E0 48 89 45 E8 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 4C 8D 86 ?? ?? ?? ??")]
+    public partial CStringPointer FormatAddonText2IntIntUIntUIntUIntUInt(uint addonId, int intParam1, int intParam2, uint uintParam1, uint uintParam2, uint uintParam3, uint uintParam4);
```

Comment:
```suggestion
```

No xrefs and a giant signature that detects multiple functions when scanning in IDA